modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl

```diff
@@ -1179,6 +1157,84 @@ that may or may not be enforced with a unique index at the database level. Case
 					</#if>
 				</#list>
 
+				<#if finder.hasColumn("groupId") && finder.getColumn("groupId").hasArrayableOperator()>
+
+					<#list finderColsList as finderCol>
+						<#if finderCol.name == "groupId">
+
+							List<Long> enabledGroupIdsList = new ArrayList<Long>();
+							List<Long> notEnabledGroupIdsList = new ArrayList<Long>();
+
+							for (${finderCol.type} ${finderCol.name} : ${finderCol.names}) {
+								if (!InlineSQLHelperUtil.isEnabled(${finderCol.name})) {
+									notEnabledGroupIdsList.add(${finderCol.name});
+								}
+								else {
+									enabledGroupIdsList.add(${finderCol.name});
+								}
+							}
+
+							List<${entity.name}> list = new ArrayList<${entity.name}>();
```

Comment:
As far as variable naming, `list` is not going to get approved by Brian. Maybe something like `${entity.name}s`